### DISCUSSION TITLE
Custom modules not appearing in Configuration=>Features

### DISCUSSION THREAD

**Discussion by sarpvahabi** [OrchardCMS/OrchardCore#11266] [Q&A]
Hello, I try to add an existing module to the latest version (.NET 6). For instance, I cloned Etch.OrchardCore.Gallery from github and changed its target framework to .NET 6 in the .csproj file, and also changed the references to proper project references. I also added the project reference to the main application. But when I start the application, admin=> configuration=> features: there is no Gallery feature to enable. Thanks in advance for the help.
![Adsız](https://user-images.githubusercontent.com/19488895/155718881-f823e077-b0ab-4453-bda6-850794727d68.png)

**Comment by hishamco** [comment]
Please be sure you are searching for a module or feature exist in your `Manifest.cs`

**Reply by sarpvahabi** [reply]
Hello, thanks for your reply, and yes modules have their own Manifest.cs. I am searching with the Name of the module. I try also with a couple of cloned modules (which I changed their target framework to .Net 6 and none of them appears in the Features list.

![image](https://user-images.githubusercontent.com/19488895/155727646-6e9f069c-edbc-4a09-b410-3f993bb2e372.png)

**Comment by hishamco** [comment]
Strange!! I didn't see such issue except if I miss the referense the module to the host application or use invalid target in the module csproj

**Reply by ns8482e** [reply]
Yes adding reference to main application is required to be listed in Feature's list
